cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -273,6 +273,23 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     )
   }
 
+  @Test def showConstraints() {
```

Comment:
This is not rendered at all, I think you need to include it in the file operators.asciidoc

Reply:
yes I do